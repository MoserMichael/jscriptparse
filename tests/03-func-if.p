def foo(val) {
    if val < 12
        println("should be happy years")
    elif val <= 30
        println("youth age")                
    else {
        println("after youth age")
    }
}
foo(6)
foo(20)
foo(42)
