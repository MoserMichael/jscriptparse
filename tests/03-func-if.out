should be happy years
youth age
after youth age
